src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -486,8 +487,23 @@ void RippleAddress::setAccountPublic (RippleAddress const& generator, int seq)
 }
 
 bool RippleAddress::accountPublicVerify (
-    uint256 const& uHash, Blob const& vucSig, ECDSA fullyCanonical) const
+    Blob const& message, Blob const& vucSig, ECDSA fullyCanonical) const
```

Comment:
It appears that our ed25519 library always generates canonical signatures.  I added an assertion that this is so, as well as a check on verification.